docs/updates.md

```diff
@@ -2,6 +2,10 @@
 
 Here are some updates we've made to our autograder platform. If you have any questions or issues with any of these changes, please email [help@gradescope.com](mailto:help@gradescope.com).
 
+## Mar 22th, 2022
+
+We have added rich formatting options to the autograder output!
+See [our documentation on the formatting options here](../specs#output-string-formatting) for more information.
```

Comment:
Add a newline here to break things up
```suggestion
See [our documentation on the formatting options here](../specs#output-string-formatting) for more information.

```

In some markdown parsers e.g. on GitHub I've noticed that having text above a heading without a line break causes the text to become part of the heading, but it seems like it's not an issue here. Oh I see, the issue is with the underline style headings and not the octothorpe-style ones, e.g.

```
Foobar
Baz
------
```

becomes

Foobar
Baz
------